PGESnake/PGESnake.cpp

```diff
@@ -14,10 +14,8 @@ class Snake : public olc::PixelGameEngine {
 	//Game variables
 	int score;
 	//Snake variables
-	float SnakeXPos, SnakeYPos, OldSnakeXPos, OldSnakeYPos;
+	float SnakeXPos, SnakeYPos;
```

Comment:
this can be turned to ints

Reply:
This game is currently resolving movement with floats (using fElapsedTime). There would have to be some more changes to add a delay to the movement of the snake and a few other subtle changes.

That's likely why this has been a bit more challenging than a normal grid-based Snake game.